### ISSUE TITLE
VDI profile reference support

### ISSUE THREAD

**Issue by lm-sig** [F5Networks/f5-appsvcs-extension#153]
##### Do you already have an issue opened with F5 support?
No.

##### ISSUE TYPE

 - Feature Idea

##### AS3 BUILD/ VERSION

```
f5-appsvcs-3.14.0-4.noarch.rpm
```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.3
  Build       0.0.6
  Edition     Final
  Date        Fri Aug  2 20:39:15 PDT 2019
```

##### SUMMARY

3.14.0 introduced Connectivity profile support. I have a Citrix connectivity profile and it also requires a VDI Profile be attached to the Virtual Server. I do not see any way to do this with AS3.

##### STEPS TO REPRODUCE

    {
        "class": "AS3",
        "action": "deploy",
        "persist": true,
        "declaration": {
            "class": "ADC",
            "schemaVersion": "3.14.0",
            "id": "test-declaration-0",
            "label": "Test Declaration 0",
            "remark": "Test Declaration for My Apps",
            "Partition": {
                "class": "Tenant",
                "MyApp1": {
                    "class": "Application",
                    "template": "generic",
                    "MyApp1_VIP": {
                        "class": "Service_HTTPS",
                        ...
                        "profileAccess": {
                            "bigip": "/Common/MyAccess_Test"
                        },
                        "profileConnectivity": {
                            "bigip": "/Common/MyConnectivity_Test"
                        },
                        ...
                }
            }
        }
    }

##### EXPECTED RESULTS

```
VDI profile support
```

##### ACTUAL RESULTS

```
Virtual Server (/Partition/MyApp1/MyApp1_VIP) requires VDI profile (resource /Common/MyConnectivity_Test-TEST_FSD) for this assignment.
```

**Comment by dstokesf5** [comment]
I have added this feature request to our internal product backlog as AUTOTOOL-1053.

**Comment by dstokesf5** [comment]
This was resolved in the 3.24.0 release.

**Comment by lm-sig** [comment]
I have tested this and it works. Thank you!
